code/game/gamemodes/wizard/artefact.dm

```diff
@@ -198,4 +198,74 @@
 	playsound(src,'sound/weapons/inc_tone.ogg', 50, 1)
 	spawn(2 SECONDS)
 		explosion(loc, 0, 1, 2, 3)
-		qdel(src)
\ No newline at end of file
+		qdel(src)
+
+/obj/item/phylactery
+	name = "strange stone"
+	desc = "A stone, decorated with masterly crafted silver, adorned with silver in the shape of a human skull. It hums with malignancy."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "phylactery_empty_noglow"
+	var/charges = 0
+	var/soulbound
+	var/mob/bound_soul
+
+/obj/item/phylactery/attackby(obj/item/I, mob/user)
+	if(istype(I, /obj/item/device/soulstone))
+		var/obj/item/device/soulstone/S = I
+		var/mob/living/simple_animal/shade/sacrifice = locate() in S
+		if(sacrifice)
+			visible_message("<span class = 'warning'>The soul within \the [I] is released unto \the [src].</span>")
+			S.name = initial(S.name)
+			charges++
+			update_icon()
+			qdel(sacrifice)
+	else
+		..()
+
+/obj/item/phylactery/Destroy()
+	bound_soul.on_death.Remove(soulbound)
+	soulbound = null
+	bound_soul = null
+	..()
+
+
+/obj/item/phylactery/update_icon()
+	if(soulbound)
+		if(charges >= 1)
+			icon_state = "phylactery"
+		else
+			icon_state = "phylactery_empty"
+	else
+		icon_state = "phylactery_empty_noglow"
+
+/obj/item/phylactery/attack_self(mob/user)
+	if(!soulbound && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/datum/organ/external/E = H.get_active_hand_organ()
+		if(locate(/datum/wound) in E.wounds)
```

Comment:
eh feel free go ignore this tiny little thing, but it'd be better if this was an organ-level proc, for instance ``if(E.is_wounded())``